tests/iostability/test_iostability_degraded_path.py

```diff
@@ -68,23 +68,41 @@ def setup_class(cls):
         cls.s3t_obj = S3TestLib()
         cls.iolib = IOStabilityLib()
         cls.test_completed = False
+        cls.duration_in_days = os.getenv("DURATION_OF_TEST_IN_DAYS",
+                                         cls.test_cfg['happy_path_duration_days'])
+
+        cls.clients = os.getenv("CLIENT_SESSIONS_PER_WORKER_NODE", 0)
+        if cls.clients == 0:
+            if cls.setup_type == 'HW':
+                cls.clients = cls.test_cfg['sessions_per_node_hw']
+            else:
+                cls.clients = cls.test_cfg['sessions_per_node_vm']
+        cls.mail_id = os.getenv("MAIL_ID", None)
+        cls.start_time = datetime.datetime.now()
+        cls.mail_notify = MailNotification(sender="priyanka.borawake@seagate.com",
+                                           receiver=cls.mail_id)
+        cls.mail_notify.mail_thread.start()
 
     def teardown_method(self):
         """Teardown method."""
+        self.log.info("Teardown method")
         if not self.test_completed:
             self.log.info("Test Failure observed, collecting support bundle")
+            self.mail_notify.event_fail.set()
             path = os.path.join(LOG_DIR, LATEST_LOG_FOLDER)
             resp = support_bundle_utils.collect_support_bundle_k8s(local_dir_path=path,
                                                                    scripts_path=K8S_SCRIPTS_PATH)
             assert_utils.assert_true(resp)
+        else:
+            self.mail_notify.event_pass.set()
 
     @pytest.mark.lc
     @pytest.mark.io_stability
     @pytest.mark.tags("TEST-40172")
     def test_object_crud_single_pod_failure(self):
         """Perform Object CRUD operations in degraded mode in loop using S3bench for 7 days."""
```

Comment:
The comment also needs to be inline with new change of %s days instead of "7"

Reply:
Should this be changed to n ? Default is 7 days. 
User may enter other values as per requirement.

Reply:
@priyankaborawake , yes make it `n` to make it generic.